crates/biome_js_formatter/src/js/classes/method_class_member.rs

```diff
@@ -193,19 +194,6 @@ impl FormatAnyJsMethodMember {
     }
 }
 
-declare_node_union! {
-     AnyMemberName = AnyJsClassMemberName | AnyJsObjectMemberName
-}
```

Comment:
I took the opportunity of removing this type because it is a duplicate of `AnyJsMemberName`